jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -431,7 +430,7 @@ void java_bytecode_convert_methodt::convert(
   // to the symbol later.
   java_method_typet method_type =
     to_java_method_type(symbol_table.lookup_ref(method_identifier).type);
-  method_type.set_is_final(m.is_final);
+  method_type.set(ID_C_class, class_symbol.name);
```

Comment:
Suspect this is a mis-merge; this is now done by `set_declaring_class` above.

Reply:
Fixed!